src/Hooks/TransformSynthesizers.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MarcoRieser\Livewire\Hooks;
+
+use Livewire\ComponentHook;
+use MarcoRieser\Livewire\Synthesizers\TransformableSynthesizer;
+
+class TransformSynthesizers extends ComponentHook
+{
+    public function render($view, $data): void
+    {
+        if (! config('statamic-livewire.synthesizers.enabled', false) || ! config('statamic-livewire.synthesizers.transform', true)) {
+            return;
+        }
+
+        collect($data)
+            ->map(function ($value) {
+                $synthesizer = $this->getMatchingSynthesizer($value);
+
+                return $synthesizer ? call_user_func([$synthesizer, 'transform'], $value) : $value;
+            })
+            ->each(fn ($value, $key) => $view->with($key, $value));
```

Comment:
Miiiight be nicer, `with` takes an array?

![CleanShot 2025-04-25 at 14 12 04@2x](https://github.com/user-attachments/assets/1a75e8ec-86d4-40b6-b141-bd8c27fd38d3)